iceoryx2-ffi/ffi/src/tests/node_name_tests.rs

```diff
@@ -29,12 +29,12 @@ fn basic_node_name_test() -> Result<(), Box<dyn std::error::Error>> {
         assert_that!(ret_val, eq(IOX2_OK));
 
         let mut node_name_len = 0;
-        let node_name_c_str = iox2_node_name_as_c_str(
+        let node_name_c_ptr = iox2_node_name_as_str(
```

Comment:
@elBoberido The name `node_name_c_ptr` causes me to scratch my head on first read. It's because it is just a pointer to chars (i.e. not null terminated)?

Reply:
@elBoberido Maybe `node_name_chars` ?

Reply:
Should the function then be named `iox2_node_name_as_chars`?

Reply:
@elBoberido Works for me

Reply:
I liked the function name since `str` has this implicit null-termination in the C world and `chars` is great for just a char array.